FrostyModSupport/FrostyModExecutor.cs

```diff
@@ -362,7 +362,7 @@ private void ProcessModResources(IResourceContainer fmod)
 
                             modifiedEbx.TryAdd(entry.Name, entry);
                             if (!archiveData.ContainsKey(entry.Sha1))
-                                archiveData.GetOrAdd(entry.Sha1, new ArchiveInfo() { Data = data, RefCount = 1 });
+                                archiveData.TryAdd(entry.Sha1, new ArchiveInfo() { Data = data, RefCount = 1 });
```

Comment:
the try add is a bool so you can put it in the if statement

Reply:
i just made it the same as for chunks and res

Reply:
i just made it the same as it already is for res and chunks